jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -0,0 +1,144 @@
+lib = library(identifier: "jenkins@20211118", retriever: legacySCM(scm))
+
+pipeline {
+    options {
+        timeout(time: 3, unit: 'HOURS')
+    }
+    agent none
+    environment {
+        AWS_ROLE_ARN = "arn:aws:iam::${AWS_ACCOUNT_PUBLIC}:role/opensearch-test"
```

Comment:
Where is `AWS_ROLE_ARN` and `AWS_ROLE_SESSION_NAME` being used?

Reply:
Looks the original attention when added to OpenSearch integration test was to have it binded and used for the uploadTestResults. But we currently hardcode the role and session so I went ahead and deleted it from both Jenkinsfile.